ui/admin/app/controllers/scopes/scope/roles/role/manage-scopes/index.js

```diff
@@ -14,6 +14,12 @@ export default class ScopesScopeRolesRoleManageScopesIndexController extends Con
 
   @service router;
 
+  // =attributes
+
+  queryParams = ['showCheckIcon'];
```

Comment:
I'm wondering if you need this queryParam in the first place. It feels kind of out of place since it's being used to just pass a flag to know if something should be displayed after saving. (Not sure how you feel about it, I could be convinced)

It looks like you only set it in the controllers on transition, is it possible to just set a property on the parent controller instead by using `@controller` and resetting when necessary?

Reply:
Great point, I updated to be just a property instead!